### PR TITLE
Quickly add  endpoint to return the list of all of the countries

### PR DESCRIPTION
**Pull request by csabadenes**

### CODE REVIEW

File: EquiTrack/EquiTrack/urls.py [hunk 1]
```diff
@@ -126,6 +126,7 @@
     url(r'^api/audit/', include('audit.urls', namespace='audit')),
     url(r'^api/v2/', include('reports.urls_v2')),
     url(r'^api/v2/', include('partners.urls_v2', namespace='partners_api')),
+    url(r'^api/v2/', include('users.urls_v2', namespace='workspaces_api')),
```
Comment (robertavram) [inline EquiTrack/EquiTrack/urls.py]: we can remove this.. can you make another PR? :)

Comment (csabadenes) [reply EquiTrack/EquiTrack/urls.py]: Ofc, i have already changed this, i'll make another PR
